### PR TITLE
feat(base): Move to  Conventional Commits for versioning.

### PR DESCRIPTION
**Pull request by seanhealy**
## Motivations

As a first step towards automating our Atlantis releases this moves us to generating release versions based on the PRs that have been merged since the last release.

### Added

- Now uses [Conventional Commits](https://github.com/lerna/lerna/tree/master/commands/version#--conventional-commits) to auto calculate versions when creating releases. This is a first step to allow us to do auto deploying of Atlantis.

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

- Run `npm run release-the-kraken`
- Note the prompt confirming you wish to make a release contains a generated version.
- Select `no` you do not wish to create a release as we're just checking that the version is generated here.

image

## Notes

Notes for when we want to move publishing to CI.

- `https://github.com/lerna/lerna/issues/2022`
- https://github.com/lerna/lerna/tree/master/commands/version#--git-remote-name
- https://github.com/lerna/lerna/tree/master/commands/version#--allow-branch-glob
- https://github.com/lerna/lerna/tree/master/commands/version#--amend

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

**Comment by seanhealy** [comment]
Notes for when we want to move publishing to CI.

- https://github.com/lerna/lerna/issues/2022
- https://github.com/lerna/lerna/tree/master/commands/version#--git-remote-name
- https://github.com/lerna/lerna/tree/master/commands/version#--allow-branch-glob
- https://github.com/lerna/lerna/tree/master/commands/version#--amend

**Comment by jessevanherk** [comment]
@seanhealy shouldn't the commit on this branch be a conventional commit?

**Comment by jessevanherk** [comment]
@seanhealy The PR description is quite light here - I'm not entirely sure what this PR actually changes. What's the before vs after on it, and what does it unblock?

**Comment by seanhealy** [comment]
@jessevanherk Updated.

**Comment by jessevanherk** [comment]
@seanhealy I pulled down the branch, but I have no idea how to QA this. Please add some instructions for non-atlanteans.

**Comment by jessevanherk** [comment]
I ran the release-the-kraken task, and confirmed that it ran lerna and was working on bumping the version.

image

LGTM

File: lerna.json [hunk 1]
```diff
@@ -1,6 +1,12 @@
...
+  "version": "independent",
+  "packages": ["packages/*"],
+  "command": {
+    "version": {
+      "conventionalCommits": true
+    },
+    "publish": {
+      "ignoreChanges": ["package-lock.json"]
+    }
+  }
```
Comment (jessevanherk) [inline lerna.json]: interesting that it doesn't ignore package-lock by default. already LGTMed in another comment, but still LGTM.

Comment (seanhealy) [reply lerna.json]: Oh, I may need to relative path that.

Comment (seanhealy) [reply lerna.json]: Just pushed a fix for this.

Comment (jessevanherk) [reply lerna.json]: took this for a spin with a dummy commit to package-lock.json. lerna was cool with it.

**Pull request merged**